applet: use symbolic nvidia/intel icons

NikoKrause opened the pull request:
make intel/nvidia icons bigger, else they look too small as symbolic ones

Since Cinnamon 3.6 the current icons look squashed/squared. Changed the svg icons to fix this issue:

Fixes https://github.com/linuxmint/Cinnamon/issues/7086
Closes https://github.com/linuxmint/cinnamon-spices-applets/issues/1552

brownsr commented:
Sorry, doesn't work when I tried it.  Still gives strange aspect ratios and sizes.  The on_panel_height_changed stuff seems to be causing that.  Just strip it out and it works.  That leaves one further issue, that the svgs don't seem to scale as large as you would want them in a large panel size, that may be a wider issue with our handling of svgs.  It's probably liveable with if you really want symbolics.  Otherwise, that can readily be fixed by using different icons - you already have two perfectly good icons designed for the systray in the pixmap directory, setting the icons to use these  [ prime-tray-intel, prime-tray-nvidia] works fine and seems to work on any panel size, and then you can dump the additional icons entirely.  I just used set_applet_icon_name ("prime-tray-nvidia") etc. here.  With the move over to IconApplet you can dispense with the getDisplayLayout call entirely.

One more thing I noticed, which is pre-existing, is that clicking the applet to pop up a one-entry menu, which you then click to call the settings program is really a bit odd.  I can't see a reason why it can't just call the settings program directly.

Regards

Simon

NikoKrause commented:
Hey,

when did you tried it? I have rebased the PR yesterday, because there was a typo in it. Maybe you tried the pre-rebased version?

I first wanted to add colored versions, because the symbolics are too small if you don't resize them. But if you use a blue theme you barely see the intel icon.

The one-entry menu can probably be changed. I don't have a machine where I can test this applet, therefore I just changed the icons.
